resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -104,20 +104,23 @@
       experiment='{% for experiment in cuffquant %}{{ experiment.cxb.file }},{% endfor %}'
       replicates='{% for r in replicates %}{{r}},{% endfor %}'
       labels='{% for l in labels %}{{l}},{% endfor %}'
+      sample_names='{% for sample in cuffquant %}{{sample|sample_name}},{% endfor %}'
```

Comment:
`cuffquant` is list of cuffquant objects, not samples. So using `sample` in `{% for sample in cuffquant %}` is misleading. Use `cq`, or something similar, instead of `sample`.